FDTD/FDTD.f90

```diff
@@ -352,15 +352,27 @@ subroutine dt_evolve_Ac_1d
   integer :: ix_m
   real(8) :: RR(3) ! rot rot Ac
 
-  Ac_old_m=Ac_m
-  Ac_m=Ac_new_m
+!$omp parallel do default(none) &
+!$    private(ix_m) &
+!$    shared(NXvacL_m,NXvacR_m,Ac_m,Ac_old_m,Ac_new_m)
+  do ix_m=NXvacL_m,NXvacR_m
```

Comment:
We need to extend the range of do loop, `do ix_m=NXvacL_m-1,NXvacR_m+1`, because the size of these arrays are slightly larger than the present range.